ArmPkg/Library/ArmMonitorLib/AArch64/ArmMonitorLib.masm

```diff
@@ -7,7 +7,6 @@
 //
 
     AREA |.text|, CODE, READONLY
-    ALIGN 3

```

Comment:
@kuqin12 what is the right ALIGN value for armasm files? We seem to have a mismash of align values in the different masm files.

Reply:
Do you mean align 8 vs. align 3? I should be align 8 bytes for this type of functions. But `2^3 = 8`, but I do not know what alignment scheme masm files use.

Reply:
![image](https://github.com/user-attachments/assets/8dd6292a-9c52-487f-a4a9-1c68d5ea235f)

![image](https://github.com/user-attachments/assets/05717164-a1eb-47fc-9672-f8b57445eb6c)

Reply:
![image](https://github.com/user-attachments/assets/a649c2fb-15d3-4d1f-aa8a-1ed612c26fe1)

So `ALIGN 3` should be in the AREA directive here.